src/Test/Utilities/SharedCompilationUtils.cs

```diff
@@ -61,7 +61,14 @@ internal static CompilationTestData.MethodData GetMethodData(this CompilationTes
             [CallerLineNumber]int expectedValueSourceLine = 0,
             [CallerFilePath]string expectedValueSourcePath = null)
         {
+            const string moduleNamePlaceholder = "{#Module#}";
             string actualIL = GetMethodIL(method);
+            if (expectedIL.IndexOf(moduleNamePlaceholder) >= 0)
+            {
+                var module = method.Method.ContainingModule;
+                var moduleName = Path.GetFileNameWithoutExtension(module.Name);
```

Comment:
I thought a pseudo-variable representing the module name without extension would be more useful than one that includes the extension.